Cargo.toml

```diff
@@ -9,24 +9,20 @@ license = "MPL-2.0"
 rand_core = "^0.6"
 rand_chacha = { version = "^0.3", default-features = false }
 sha2 = { version = "^0.10", default-features = false }
-phoenix-core = { version = "0.16.0-rc", features = ["canon"] }
-dusk-pki = { version = "0.10.0-rc", features = ["canon"] }
+phoenix-core = { version = "0.18", default-features = false, features = ["alloc", "rkyv-impl"] }
+dusk-pki = { version = "0.11", default-features = false }
 dusk-bytes = "^0.1"
-dusk-schnorr = { version = "0.10.0-rc", default-features = false }
-dusk-jubjub = { version = "0.11", default-features = false }
-dusk-poseidon = { version = "0.25.0-rc", features = [
-  "canon",
-], default-features = false }
-dusk-plonk = { version = "0.10", default-features = false }
-rusk-abi = "0.7"
-canonical = "0.7"
-dusk-bls12_381-sign = { version = "0.3.0-rc", default-features = false, features = [
-  "canon",
-] }
+dusk-schnorr = { version = "0.12", default-features = false }
+dusk-jubjub = { version = "0.12", default-features = false }
+dusk-poseidon = { version = "0.29.1-rc.0", default-features = false }
+dusk-merkle = { version = "0.4.1-rc.0", features = ["poseidon", "rkyv-impl"] }
+dusk-plonk = { version = "0.13", default-features = false }
+rusk-abi = { version = "0.8.0-piecrust.0.5", features = ["host"] }
```

Comment:
@ureeves is this still required?

cc @ZER0

Reply:
The "host" feature will cease to be required when I complete the port of `rusk-abi` to `piecrust@0.6`

Reply:
Perfect!